packages/notifi-frontend-client/lib/client/ensureSource.ts

```diff
@@ -713,22 +704,32 @@ const getHealthCheckFilter = (
     inputs,
   );
 
-  let threshold = 0;
-  let thresholdDirection: ThresholdDirection = 'below';
+  // Default: checkRatios[0]
+  let threshold = checkRatios[0].ratio;
+  let thresholdDirection = checkRatios[0].type;
 
-  const checkInputsCategory = (
-    inputs: HealthCheckInputs,
+  const checkInputsIsWithIndex = (
+    inputs: Record<string, unknown>,
   ): inputs is HealthCheckEventInputsWithIndex => {
     if ('index' in inputs) {
       return true;
     }
     return false;
   };
 
-  if (checkInputsCategory(inputs)) {
+  const checkInputsIsWithCustomPercentage = (
```

Comment:
I think these can be declared outside of the `getHealthCheckFilter` function

Reply:
Moved the checker out